pleskdistup/actions/distupgrade.py

```diff
@@ -110,6 +119,10 @@ def estimate_revert_time(self) -> int:
         return 20
 
 
+SetupDebianRepositories = SetupAptRepositories
+SetupUbuntuRepositories = SetupAptRepositories
```

Comment:
Nit: It seems that we can remove it and just use `SetupAptRepositories`. However, this means we will need to update the actions for ubuntu18to20 and debian11to12 when we move them to a new version of the submodule.

Reply:
This is a precaution if we would need to make them different classes in future. Then `SetupAptRepositories` would be the base class for `SetupDebianRepositories`, `SetupUbuntuRepositories`.